src/moveit_pro_kinova_configs/kinova_gen3_mujoco_config/objectives/demo_trajectory_stitching.xml

```diff
@@ -0,0 +1,144 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<root BTCPP_format="4" main_tree_to_execute="Demo Trajectory Stitching">
```

Comment:
I'd be interested in a better name that tells a story. I think @MikeWrock has some ideas after chatting with him today